packages/react-native/src/Authenticator/Defaults/ConfirmResetPassword/__tests__/__snapshots__/ConfirmResetPassword.spec.tsx.snap

```diff
@@ -74,7 +74,7 @@ Array [
         style={
           Object {
             "alignItems": "center",
-            "borderColor": "hsl(210, 8%, 55%)",
+            "borderColor": "hsl(210, 10%, 58%)",
```

Comment:
There are a bunch of snapshot test updates because I changed a border color.

Reply:
👍